util/admin/admin.go

```diff
@@ -91,6 +91,12 @@ func isJobRollbackable(job *model.Job, id int64) error {
 			job.SchemaState == model.StateDeleteReorganization {
 			return ErrCannotCancelDDLJob.GenWithStackByArgs(id)
 		}
+	case model.ActionDropColumn:
+		if job.SchemaState == model.StateDeleteOnly || job.SchemaState == model.StateNone ||
+			job.SchemaState == model.StateWriteOnly ||
```

Comment:
Why this state can't cancel job?

Reply:
@zimulala Because columnInfo has been modified since the `public -> write only `stage.

Reply:
job.SchemaState == model.StateNone，this will make can't cancel drop column before the job first run?